### DISCUSSION TITLE
Dismissed dependabot alert for ws

### DISCUSSION THREAD

**Discussion by bryevdv** [bokeh/bokeh#11313] [General]
ref: https://github.com/advisories/GHSA-6fc8-4gx4-v693

Just noting for transparency that I dismissed this alert without action:

* we control the contents of the Sec Protocol header
* the ws server is in Python, not in node, so it is not relevant to us in any case

**Comment by mattpap** [comment]
I'm in the process of upgrading bokehjs' dependencies, so this will get updated anyway.
